subsys/bluetooth/host/CMakeLists.txt

```diff
@@ -108,6 +108,15 @@ if(CONFIG_BT_CONN_DISABLE_SECURITY)
     )
 endif()
 
+if(CONFIG_BT_USE_PSA_API)
+    zephyr_library_link_libraries_ifdef(CONFIG_MBEDTLS mbedTLS)
+    if (CONFIG_BUILD_WITH_TFM)
```

Comment:
NIT: zephyr_library_include_directories_ifdef can be used there instead to some small code cleanup

zephyr_library_include_directories is using PRIVATE as well as ${ZEPHYR_CURRENT_LIBRARY}